.env.example

```
@@ -5,5 +5,6 @@ OPEN_WEBUI_URL=http://localhost:3000
 OPEN_WEBUI_MODEL=gpt-oss-120b
 OPEN_WEBUI_PROMPT="This user message is sent by line chat. In other word, you are playing the role of line bot. Combine the exist system prompts and give concise reply like other line bot."
 STORAGE_PATH="./files"
+LOG_PATH="./logs"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)